test-cypress/cypress/e2e/2-advanced-examples/waiting.cy.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+context('Waiting', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/waiting');
+  });
+  // BE CAREFUL of adding unnecessary wait times.
+  // https://on.cypress.io/best-practices#Unnecessary-Waiting
+
+  // https://on.cypress.io/wait
+  it('cy.wait() - wait for a specific amount of time', () => {
+    cy.get('.wait-input1').type('Wait 1000ms after typing');
+    cy.wait(1000);
+    cy.get('.wait-input2').type('Wait 1000ms after typing');
+    cy.wait(1000);
+    cy.get('.wait-input3').type('Wait 1000ms after typing');
+    cy.wait(1000);
```

Comment:
This is a third instance of a static wait. Try to avoid using fixed waits and opt for more reliable methods like checking for element visibility or network request completion.